src/main/scala/net/akmorrow13/endive/utils/Window.scala

```diff
@@ -82,6 +85,14 @@ case class Window(tf: TranscriptionFactors.Value,
   }
 }
 
+
+case class FeaturizedLabeledWindow(labeledWindow: LabeledWindow, features: DenseVector[Double]) {
+
```

Comment:
I don't like that we have 2 case classes that do the same thing. I think we should just keep the old one for simplicity. see https://github.com/akmorrow13/endive/blob/a513f5d7977c2278555db18dbf924dbc728f689d/src/main/scala/net/akmorrow13/endive/processing/Preprocess.scala#L403